internal/tofu/graph_walk_context.go

```diff
@@ -58,10 +57,8 @@ type ContextGraphWalker struct {
 	variableValues     map[string]map[string]cty.Value
 	variableValuesLock sync.Mutex
 	providerCache      map[string]providers.Interface
-	providerSchemas    map[string]providers.ProviderSchema
```

Comment:
Tiny nitpick: would you mind making this change in another PR? then it will be easier to review/discuss/revert if needed. Thanks 👍

Reply:
Done!